src/components/UserInfo.js

```diff
@@ -1,18 +1,25 @@
 export default class UserInfor {
-  constructor({ nameSelector, aboutMeSelector }) {
+  constructor({ nameSelector, aboutSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
-    this._aboutMeElement = document.querySelector(aboutMeSelector);
+    this._aboutElement = document.querySelector(aboutSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
     return {
-      profileName: this._nameElement.textContent,
-      profileTitle: this._aboutMeElement.textContent,
+      name: this._nameElement.textContent,
+      about: this._aboutElement.textContent,
+      avatar: this._avatarElement.src,
     };
   }
 
-  setUserInfo(data) {
-    this._nameElement.textContent = data.profileName;
-    this._aboutMeElement.textContent = data.profileTitle;
+  setUserInfo(userData) {
+    this._nameElement.textContent = userData.name;
+    this._aboutElement.textContent = userData.about;
+    this._avatarElement.src = userData.avatar;
```

Comment:
Needs correction: Please remove this line and use ` setUserAvatar` method in index.js instead, because the avatar is controlled by a different form. When initializing the user you should call both methods.